common/app/layout/Sublink.scala

```diff
@@ -329,77 +330,21 @@ object ContentCard {
 
   def fromApiContent(apiContent: contentapi.Content): Option[ContentCard] = {
 
-    apiContent.fields.map { fields =>
+    val cardTypesForRecommendations = ItemClasses(mobile = MediaList, tablet = Standard, None)
```

Comment:
desktop defaults to None, so isn't needed.  If you want to keep it, my preference is that named parameters should be all or nothing.

`ItemClasses(mobile = MediaList, tablet = Standard, desktop = None)`
or
`ItemClasses(mobile = MediaList, tablet = Standard)`